apps/documents/models.py

```diff
@@ -0,0 +1,69 @@
+import logging
+
+from django.conf import settings
+from django.core.validators import MinValueValidator, MaxValueValidator
+from django.db import models
+from enumfields import Enum
+from enumfields import EnumIntegerField
+from influxdb_metrics.loader import log_metric
+
+from apps.shipments.models import Shipment
+from apps.utils import random_id, get_s3_client
+
+LOG = logging.getLogger('transmission')
+
+
+class DocumentType(Enum):
+    BOL = 0
+    IMAGE = 1
+    OTHER = 2
+
+
+class FileType(Enum):
+    """
+    At the moment we do support just these three types
+    """
+    PDF = 0
+    JPEG = 1
+    PNG = 2
+
+
+class UploadStatus(Enum):
+    PENDING = 0
+    COMPLETED = 1
+    FAILED = 2
+
+
+class Document(models.Model):
+    id = models.CharField(primary_key=True, default=random_id, max_length=36)
+    name = models.CharField(max_length=36, null=True, blank=True)
```

Comment:
I think `name` should probably be null=False and blank=False, it can default to the uploaded filename